Make stat20data library available in shared path.

ryanlovett opened the pull request.

ryanlovett committed:
Make stat20data library available in shared path.

ryanlovett committed:
Update haven.

Version 2.3.1 is unavailable.

yuvipanda commented:
@ryanlovett ah, hmm - given there are 600 students, I think a separate hub might actually be cleaner than this. What do you think?

ryanlovett commented:
@yuvipanda I think using a shared directory is best for how the instructor wants to update the library, but let me know your concerns regarding the class size. Maybe we can chat about this during the strategy meeting.

ryanlovett committed:
Link to package in shared directory.

I'm pivoting to a new hub, so this PR probably won't go anywhere. But I
wanted to push the last commits just in case.

ryanlovett commented:
This was abandoned in favor of a new stat20 hub.
